### PR TITLE
fix(single-supporter): move cc license to end of article

### PR DESCRIPTION
**Pull request by Jasmine-liang**
Changes:
- Represent new single supporter UI on SupporterWidget;
- move cc license to end of article;
- Fix SupporterList Dialog copy;

### CODE REVIEW

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 1]
```diff
@@ -107,24 +107,39 @@ const Donators = ({ article, showAvatarAnimation = false }: DonatorsProps) => {
...
+                  zh_hant="第一個支持了這篇作品"
+                  zh_hans="第一个支持了这篇作品"
+                  en="was the first to support this article"
                 />
-              </TextIcon>
-            </button>
+              </span>
+            )}
+
+            {donatorsCount !== 1 && (
```
Comment (tx0c) [inline src/views/ArticleDetail/SupportWidget/Donators/index.tsx:123]: why not `donatorsCount > 1`

File: src/components/Dialogs/DonatorsDialog/Content.tsx [hunk 2]
```diff
@@ -120,8 +120,8 @@ const DonatorsDialogContent = ({
       <Dialog.Header
         title={
           <Translate
-            zh_hant={`${totalCount} 人支持了作品`}
-            zh_hans={`${totalCount} 人支持了作品`}
+            zh_hant={`${totalCount} 人支持過作品`}
+            zh_hans={`${totalCount} 人支持过作品`}
```
Comment (tx0c) [inline src/components/Dialogs/DonatorsDialog/Content.tsx:124]: does it have `en` title? how about move into `src/common/enums/text.ts` ?

File: src/views/ArticleDetail/index.tsx [hunk 3]
```diff
@@ -288,34 +287,27 @@ const BaseArticleDetail = ({
...
           {article?.summaryCustomized && (
             <CustomizedSummary summary={summary} />
           )}
-
           <Content
             article={article}
             content={content}
             translating={translating}
-          />
+          />{' '}
```
Comment (tx0c) [inline src/views/ArticleDetail/index.tsx:297]: remove the unnecessary whitespace `{' '}` ?

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 4]
```diff
@@ -107,24 +107,39 @@ const Donators = ({ article, showAvatarAnimation = false }: DonatorsProps) => {
...
-              aria-haspopup="true"
-            >
-              <span className="count">{donatorsCount}</span>
-              <TextIcon
-                icon={<IconArrowRight16 size="xs" />}
-                textPlacement="left"
-              >
+            {donatorsCount === 1 && (
+              <span>
+                <span className="donator-name">
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/Donators/index.tsx:112]: It's underlined, so I think it should be clickable, perhaps we can alter `<UserDigest.Plain>` and support underlined style.

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/views/ArticleDetail/SupportWidget/Donators/gql.ts [hunk 5]
```diff
@@ -17,6 +17,8 @@ export const fragments = {
             ... on User {
               id
               ...AvatarUser
+              displayName
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/Donators/gql.ts:20]: `...UserDigestPlainUser`?

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 6]
```diff
@@ -107,24 +108,46 @@ const Donators = ({ article, showAvatarAnimation = false }: DonatorsProps) => {
...
-              <span className="count">{donatorsCount}</span>
-              <TextIcon
-                icon={<IconArrowRight16 size="xs" />}
-                textPlacement="left"
+            {donatorsCount === 1 && (
+              <section className="footer">
+                <span className="donator-name">
+                  <UserDigest.Plain
+                    user={donators[0]}
+                    hasUnderline={true}
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/Donators/index.tsx:116]: can omit `={true}`

**Review by robertu7 — APPROVED** [review]
LGTM

File: src/components/UserDigest/Plain/index.tsx [hunk 7]
```diff
@@ -13,10 +13,18 @@ export type UserDigestPlainProps = {
...
   onClick?: () => void
 }
 
-const Plain = ({ user, disabled, onClick }: UserDigestPlainProps) => {
+const Plain = ({
+  user,
+  disabled,
+  onClick,
+  hasUnderline,
+  subStringDisplayNameIndex,
```
Comment (robertu7) [inline src/components/UserDigest/Plain/index.tsx:26]: displayNameLimit?

File: src/components/UserDigest/Plain/index.tsx [hunk 8]
```diff
@@ -27,11 +35,19 @@ const Plain = ({ user, disabled, onClick }: UserDigestPlainProps) => {
...
+  })
+
   return (
     <LinkWrapper {...path} disabled={disabled} onClick={onClick}>
       <section className={containerClasses}>
-        <span className="name">{user.displayName}</span>
-
+        <span className={displayNameClasses}>
+          {subStringDisplayNameIndex &&
+            makeTitle(user.displayName!, subStringDisplayNameIndex)}
```
Comment (robertu7) [inline src/components/UserDigest/Plain/index.tsx:48]: maybe we should rename `makeTitle`

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 9]
```diff
@@ -107,24 +108,46 @@ const Donators = ({ article, showAvatarAnimation = false }: DonatorsProps) => {
...
-                />
-              </TextIcon>
-            </button>
+                <span className="count">{donatorsCount}</span>
+                <TextIcon
+                  icon={<IconArrowRight16 size="xs" />}
+                  textPlacement="left"
+                >
+                  <Translate
+                    zh_hant="個人支持過本文・看全部"
```
[the remark this answers is not in the document]
Comment (tx0c) [reply src/views/ArticleDetail/SupportWidget/Donators/index.tsx:144]: `3 個人支持過本文`
vs
`3 人支持過本文` ?
